.circleci/install_zephyr_env.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Is there a specific reason to separate installing build requirement from Dockerfile rather than unify it? I suggest that you could integrate this script in your Dockerfile.